core/tests/test_utils.py

```diff
@@ -483,12 +483,54 @@ def get_expected_logout_url(self, slug):
         """Returns the expected logout URL."""
         return current_user_services.create_logout_url(slug)
 
+    def get_response(self, url, params=None, expect_errors=False,
+                     expected_status_int=200,
+                     expected_content_type='text/html'):
+        """Get a response, transformed to a Python object."""
+        response = self.testapp.get(
+            url, params, expect_errors=expect_errors,
+            status=expected_status_int)
+
+        # Testapp takes in a status parameter which is the expected status of
+        # the response. However this expected status is verified only when
+        # expect_errors=False. For other situations we need to explicitly check
+        # the status.
+        # Reference URL:
+        # https://github.com/Pylons/webtest/blob/
+        # bf77326420b628c9ea5431432c7e171f88c5d874/webtest/app.py#L1119 .
+        self.assertEqual(response.status_int, expected_status_int)
+        if not expect_errors:
+            self.assertTrue(response.status_int >= 200 and
+                            response.status_int < 400)
+        else:
+            self.assertTrue(response.status_int >= 400 and
+                            response.status_int < 600)
+        self.assertEqual(
+            response.content_type, expected_content_type)
+
+        return response
+
+    def get_response_without_checking_for_errors(
+            self, url, expected_status_int_list, params=None):
+        """Get a response, transformed to a Python object and
+        checks for a list of status codes.
+        """
```

Comment:
Missing args and returns docstring. (Consider adding for other functions too, but that's optional if you did not write them in the first place.)

Reply:
done